SmartcarAuth/UIGenerator/SmartcarAuthPickerGenerator.swift

```diff
@@ -114,9 +135,19 @@ public class SmartcarAuthPickerGenerator: SmartcarAuthUIGenerator, UIPickerViewD
         Hides the picker, invisButton, and toolBar
     */
     @objc func hidePickerView() {
-        picker.isHidden = true
-        invisButton.isHidden = true
-        toolBar.isHidden = true
+        viewController.view.layoutIfNeeded()
+        
+        //Animate disappearance
+        UIView.animate(withDuration: 0.3, animations: {
```

Comment:
I think this is causing `testHidePickerButtonPress()` to fail due to the delay. We have turned on automated test for PRs. Can you change up the test and make another commit?